modules/post/solaris/gather/enum_packages.rb

```diff
@@ -7,32 +7,32 @@ class MetasploitModule < Msf::Post
   include Msf::Post::File
   include Msf::Post::Solaris::System
 
-
-  def initialize(info={})
-    super( update_info( info,
-        'Name'          => 'Solaris Gather Installed Packages',
-        'Description'   => %q{ Post Module to enumerate installed packages on a Solaris System},
-        'License'       => MSF_LICENSE,
-        'Author'        => [ 'Carlos Perez <carlos_perez[at]darkoperator.com>'],
-        'Platform'      => [ 'solaris' ],
-        'SessionTypes'  => [ 'shell' ]
-      ))
-
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Solaris Gather Installed Packages',
+        'Description' => %q{ Post Module to enumerate installed packages on a Solaris System},
```

Comment:
While you're at it...

Unnecessary preceding whitespace, inconsustebt capitalisation, and missing fullstop.


```suggestion
        'Description' => %q{Post module to enumerate installed packages on a Solaris system.},
```

Reply:
Sounds good; Will put that in a separate PR to keep the PR limited to what Rubocop has generated

Reply:
Separate PR created: https://github.com/rapid7/metasploit-framework/pull/17617